warp10/src/main/java/io/warp10/script/formatted/DocumentationGenerator.java

```diff
@@ -61,7 +62,7 @@
 
     List<List<List<Object>>> sig = new ArrayList<>();
     List<Object> output = new ArrayList<>();
-    for (ArgumentSpecification arg: outputs) {
+    for (ArgumentSpecification arg: Lists.reverse(outputs)) {
```

Comment:
Use Collections.reverse() instead of introducing a dependency

Reply:
Lists.reverse(), contrary to Collections.reverse(), returns a view, so that there is no side effect on the list when using this function